packages/dev/src/index.js

```diff
@@ -76,10 +76,22 @@ class Server {
       server.close()
     })
   }
+  async generateChunkForUrl(url: string): Promise<Object> {
+    const chunk = await this.generateChunk(url)
+    this.state.lastChunk[url] = chunk
```

Comment:
Last chunk should be defined as an array [here](https://github.com/natew/pundle/blob/dc05298d070e36cc5b9fccdcfb228e1419a49b88/packages/dev/src/index.js#L36) and should be added to type [here](https://github.com/natew/pundle/blob/dc05298d070e36cc5b9fccdcfb228e1419a49b88/packages/dev/types.js#L41)